dags/xlml/solutionsteam_tf_nightly_supported.py

```diff
@@ -25,22 +25,23 @@
 
 
 with models.DAG(
-    dag_id="tf_latest_supported",
+    dag_id="tf_nightly_supported",
     schedule=SCHEDULED_TIME,
     tags=["solutions_team", "tf", "nightly", "supported", "xlml"],
     start_date=datetime.datetime(2023, 8, 16),
     catchup=False,
 ) as dag:
   # ResNet
   tf_resnet_v2_8 = tf_config.get_tf_resnet_config(
-      tpu_version=2,
+      tpu_version="2",
       tpu_cores=8,
       tpu_zone=vm_resource.Zone.US_CENTRAL1_C.value,
       time_out_in_min=60,
+      global_batch_size=1024,
   ).run()
 
   tf_resnet_v3_8 = tf_config.get_tf_resnet_config(
-      tpu_version=3,
+      tpu_version="3",
```

Comment:
wondering if it might make more sense to convert this into an enum?

Reply:
Yes, this makes sense. I could have a follow-up PR to update all tpu versions. HDYT?